handlers/pipeline.go

```diff
@@ -1,11 +1,12 @@
 package handlers
 
 import (
-	"github.com/gaia-pipeline/gaia/helper/stringhelper"
 	"net/http"
 	"strconv"
 	"time"
 
+	"github.com/gaia-pipeline/gaia/helper/stringhelper"
+
```

Comment:
I don't know what is wrong with `go-import` (this always happens for me too) but this looks really ugly 😒